chopsticks/xcm-transfer/kusama-assethub.test.ts

```diff
@@ -0,0 +1,25 @@
+import { query, tx } from '../../helpers/api'
+
+import { kusama } from '../../networks/polkadot'
+import { statemine } from '../../networks/statemint'
+
+import buildTest from './shared'
+
+const tests = [
+  {
+    from: 'kusama',
+    to: 'statemine',
+    name: 'KSM',
+    test: {
+      xcmPalletDown: {
+        tx: tx.xcmPallet.limitedTeleportAssets(kusama.ksm, 1e12, tx.xcmPallet.parachainV3(0, statemine.paraId)),
```

Comment:
This is not the function signature of `limitedTeleportAssets`. Where is this stuff defined?

Reply:
Here https://github.com/AcalaNetwork/e2e-tests/pull/108/files#diff-fe51f1149e20f2ecd06cde6383b42166885e45523430399b4f1cc022445292daR123

Reply:
This hardcodes in [stuff like `parents: 0`](https://github.com/AcalaNetwork/e2e-tests/pull/108/files#diff-fe51f1149e20f2ecd06cde6383b42166885e45523430399b4f1cc022445292daR130), so cannot be used to test teleporting between system parachains. I know this PR is just the CI/CD but it will need to cover a large set of test cases.

Reply:
I agree, very good point. I think we as a community can evolve the e2e test repo so it can handle far more test cases.